src/partials/layout/_footer.html

```diff
@@ -20,51 +20,59 @@
         /></picture>
       </a>
     </div>
-    <div class="footer-container">
+    <div class="footer__container">
       <nav class="footer__nav">
-        <ul class="footer__nav-list list">
-          <li class="footer__nav-item">
-            <a class="footer__nav-link link" href="./index.html"
+        <ul class="footer__nav--list list">
+          <li class="footer__nav--item">
+            <a class="footer__nav--link link" href="./index.html"
```

Comment:
all childs of nav must be elements of block footer-nav

Reply:
changed